stratum/hal/lib/common/yang_parse_tree_paths.cc

```diff
@@ -371,43 +379,48 @@ U GetValue(
 }
 
 // Optical-port-specific version.
```

Comment:
This becomes a general function. I think we can rename some variable

Reply:
Done. I left the comment about it being optical port specific to match the others.